### ISSUE TITLE
v1.9.0: Unable to switch window on Android

### ISSUE THREAD

**Issue by mayureshshirodkar** [appium/appium#11265]
## The problem

On trying to switch window, Appium server throws an error. In the script which has a Hybrid App as AUT, an in-app browser is opened, some operations are performed and it is closed. Focus returns to the current app window for further action.

On switching the window handle, there is the following error.

Error:
> org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: unknown error: 'name' must be a string(Session info: chrome=68.0.3440.91)

## Environment

* Appium version (or git revision) that exhibits the issue: 1.9.0
* Last Appium version that did not exhibit the issue (if applicable):1.8.2-Beta
* Desktop OS/version used to run Appium: Mac 10.13.5
* Mobile platform/version under test: Android 7.0
* Real device or emulator/simulator: Moto G4
* Java-Client: Latest master commit

## Link to Appium logs

https://gist.github.com/mayureshshirodkar/c23e4d3494b705d98460e91b7785d328

## Code To Reproduce Issue [ Good To Have ]

```
Set<String> handles = Setup.getInstance().getWebDriver().getWindowHandles();
for (String winHandle : handles) {
Setup.getInstance().getWebDriver().switchTo().window(winHandle);
}
```

**Comment by mayureshshirodkar** [comment]
@mykola-mokhnach is this available to test?

**Comment by mykola-mokhnach** [comment]
The change has been published in base driver v3.6.0

**Comment by sripappula** [comment]
I'm still facing the same problem..i'm on Appium base driver @3.16.0
